### PR TITLE
Homework day8 colections

### PR DESCRIPTION
**Pull request by kovtunroman**

### CODE REVIEW

File: src/homework/day8/collections/Cars.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        cars.add("Jigul");
+        cars.add("Audi");
+
+        printCarsrToFile(cars);
+        cars = findAndDeleteCarsWIthLengthMoreThenFour(cars);
+        Printer.printListIterateByIndexThrough(cars, PrinterSymbol.SPACE);
+    }
+
+    private static void printCarsrToFile(List<String> inputCars) {
+        BufferedWriter out = null;
```
Comment (siarheibalashou) [inline src/homework/day8/collections/Cars.java:28]: redundant row?

File: src/homework/day8/collections/Figures.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        for (String figure : inputFigures) {
+            if (!figure.toLowerCase().contains("i")) {
+                counter++;
+            }
+        }
+        System.out.printf("Number of figures without \"i\" char is %s", counter).println();
+    }
+
+    private static List<String> insertTriangle(List<String> inputFigures) {
+        inputFigures.set(3, "Triangle");
```
Comment (siarheibalashou) [inline src/homework/day8/collections/Figures.java]: in this row Triangle setted to 4 position in list

File: src/homework/day8/collections/Elements.java
Comment (siarheibalashou) [inline src/homework/day8/collections/Elements.java:1]: 24,25,26 rows operates with elements +1 than requested in task
image

File: src/homework/day8/collections/Countries.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+import java.util.List;
+
+public class Countries {
+    public static void main(String[] args) {
+        List<String> countries = new ArrayList<>();
+        countries.add("Andora");
+        countries.add("Portugal");
+        countries.add("England");
+        countries.add("Zambia");
+        countries.add("United Kingdom of Great Britain");
```
Comment (siarheibalashou) [inline src/homework/day8/collections/Countries.java:16]: redundant row (not according to task)
image

File: src/homework/day8/collections/Cars.java (hunk 1 above)
Comment (kovtunroman) [reply src/homework/day8/collections/Cars.java:28]: it should be like that. This needs to close out if exception happend

File: src/homework/day8/collections/Countries.java (hunk 3 above)
Comment (kovtunroman) [reply src/homework/day8/collections/Countries.java:16]: UK value was created for validation

File: src/homework/day8/collections/Elements.java
Comment (kovtunroman) [reply src/homework/day8/collections/Elements.java:1]: fixed

File: src/homework/day8/collections/Figures.java (hunk 2 above)
Comment (kovtunroman) [reply src/homework/day8/collections/Figures.java]: fixed

**Review by siarheibalashou — APPROVED** [review]

**Pull request merged**